code/amcss.c

```diff
@@ -95,7 +95,7 @@ static void report(void)
 
 static mps_addr_t make(size_t rootsCount)
 {
-  static unsigned long calls = 0;
+  static volatile unsigned long calls = 0; /* for use from debugger */
```

Comment:
m. Could refer to https://github.com/Ravenbrook/mps/commit/0c5852a0838590ea2815efa599a3e5b4ae2d5b5c and/or explain that this is to help with watchpoints.